README.md

```diff
@@ -2,19 +2,27 @@ About
 -----
 
 [![Build Status](https://travis-ci.org/astropy/ci-helpers.svg?branch=master)](https://travis-ci.org/astropy/ci-helpers)
-[![Build status](https://ci.appveyor.com/api/projects/status/4mqtucv6ks4peakf/branch/master?svg=true)](https://ci.appveyor.com/project/Astropy/ci-helpers/branch/master)
 
 This repository contains a set of scripts that are used by the
-``.travis.yml`` and ``appveyor.yml`` files of astropy packages for the
-[Travis](https://travis-ci.org) and [AppVeyor](https://www.appveyor.com/)
-services respectively.
+``.travis.yml`` file of Astropy packages for the
```

Comment:
it's not just astropy any more, but a more broad range

Reply:
Any suggested wording? 😸

Reply:
I'm not picky on words, and was just nitpicking. In fact, lets get this merged now rather than have another CI round. 
If we get bored, we can always come back and reword the readme here and possibly elsewhere, too, as it got written in a very organic way.

A more useful thing would be through to let the batchpr/issue bot loose on all the packages still using the ci-helpers appveyor, and let them know that things got evolved, they better evolve, too.